instagram/src/components/PostsPage.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+
+import SearchBar from './SearchBar';
+import dummyData from '../dummy-data';
+import Post from './Post';
+
+
+class PostPage extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            posts: dummyData,
+            likes: 0,
+            filteredPosts: []
+        }
+    }
+
+    componentDidMount() {
+        // after the initial render, CDM runs one time only
+        // perform initial data fetches here - update state with the fetched data
+        console.log('CDM running');
+
+        this.fetchPosts();
+    }
+
+    fetchPosts = () => {
+        this.setState({ posts: dummyData });
+    }
+
```

Comment:
I would delete this handleLike() function from this component and put it on your <Post/> component.